### DISCUSSION TITLE
Right clicking on a window slot!

### DISCUSSION THREAD

**Discussion by tascord** [PrismarineJS/mineflayer#2173] [Q&A] [answered]
Hello ! 
Looking to open up a window (done) and right click on slot 10!
I've been looking through the prismarine-windows docs and can't seem to find a lot...

Any help would be greatly apreciated.
Thank you!

**Comment by WhoTho** [comment, marked as answer]
Hey!

I hope this code helps

bot.on("windowOpen", (window) => {
  console.log("window open");
  //console.log(window) if you want to see the info of the window thats up
  bot.clickWindow(9, 1, 0); //slots start with the number 0 and 1 is for right click https://wiki.vg/Protocol#Click_Window
});
